### DISCUSSION TITLE
request: Add Infura RPC URLs for avalanche and avalancheFuji chains

### DISCUSSION THREAD

**Discussion by StKostyantin** [wagmi-dev/references#60] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

0.10.8

### Current Behavior

there is not enough rps infura when you use this chane

**Comment by jxom** [comment]
I believe you are talking about Infura RPC URLs? Changing the title to match. Feel free to edit if you mean something else.
